src/reference/glossary.rst

```diff
@@ -12,10 +12,21 @@ Glossary
 
       A web application used for phylogenetic visualization and analysis. :doc:`Documentation<auspice:index>`
 
+   pathogen repository
+
+      A version-controlled folder containing all files necessary to run a pathogen's :term:`workflows<workflow>`.
+
    workflow
-      also *pathogen workflow*, *pathogen analysis*, *Nextstrain workflow*
+      also *Nextstrain workflow*
```

Comment:
Commenting on this unmodified part:

https://github.com/nextstrain/docs.nextstrain.org/blob/8c7779adefeaab4851726468b98223349a5c205a/src/reference/glossary.rst#L59-L61

Suggestion: Shift from *core workflow* to *core repository* to align with the shift from *pathogen workflow* to *pathogen repository*. We maintain repos in their entirety, not just parts of the repo.

Reply:
Nice, I hadn't considered that framing. It aligns with the existing usage of the term.

Reply:
Updated in https://github.com/nextstrain/docs.nextstrain.org/pull/191/commits/1ce52a7b80e52f634c2c984736cbeaf32a36e082